test_e2e

```diff
@@ -10,6 +11,16 @@ E2E_DIRECTORY = f"{WORK_DIRECTORY}/e2e"
 #                End to end tests               #
 #################################################
 
-os.environ["PYTHONPATH"] = E2E_DIRECTORY
-os.chdir(E2E_DIRECTORY)
-subprocess.run(["pytest", ".", "-vv"])
+def main(test_filter: str = None):
+    os.environ["PYTHONPATH"] = E2E_DIRECTORY
+    os.chdir(E2E_DIRECTORY)
+    command = ["pytest", ".", "-vv"]
+    if test_filter:
+        command.extend(["-k", test_filter])
+    subprocess.run(command)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Test MAGE E2E, make sure to have your Memgraph instance ready.')
+    parser.add_argument('-k', help='Add filter to tests you want to run', type=str, required=False)
+    args = parser.parse_args()
+    main(args.k)
```

Comment:
What if we add more arguments, does it make sense to send to the main function `args` reference?

Reply:
Actually, I am not sure now, since this way we can use an easier main function as part of the external module, there is no need to send whole args reference then

Reply:
This is intended to be a standalone script, and not being used anywhere else. But the arguments could be extracted in a cleaner way

Reply:
Good point 👍🏻